pkg/model/components/kubelet.go

```diff
@@ -130,11 +138,6 @@ func (b *KubeletOptionsBuilder) BuildOptions(o interface{}) error {
 	if cloudProvider == fi.CloudProviderGCE {
 		clusterSpec.Kubelet.CloudProvider = "gce"
 		clusterSpec.Kubelet.HairpinMode = "promiscuous-bridge"
-
-		clusterSpec.Kubelet.RuntimeCgroups = "/docker-daemon"
-		clusterSpec.Kubelet.KubeletCgroups = "/kubelet"
-		clusterSpec.Kubelet.SystemCgroups = "/system"
-		clusterSpec.Kubelet.CgroupRoot = "/"
```

Comment:
We haven't shipped GCE support in kops _quite_ yet, so we don't need to worry about back-compat here.